### PR TITLE
ISPN-6055 Performance regression in replicated reads after ISPN-5465

### PR DESCRIPTION
**Pull request by danberindei**
https://issues.jboss.org/browse/ISPN-6055

### CODE REVIEW

File: core/src/main/java/org/infinispan/distribution/group/PartitionerConsistentHash.java [hunk 1]
```diff
@@ -77,7 +78,12 @@ public String getRoutingTableAsString() {
 
    @Override
    public boolean isKeyLocalToNode(Address nodeAddress, Object key) {
-      return ch.isSegmentLocalToNode(nodeAddress, getSegment(key));
+      // TODO Make this a method in ConsistentHash
```
Comment (tristantarrant) [inline core/src/main/java/org/infinispan/distribution/group/PartitionerConsistentHash.java]: Why not use a default method ?

Comment (danberindei) [inline core/src/main/java/org/infinispan/distribution/group/PartitionerConsistentHash.java]: I was hesitating about what that method should be, and I thought I was short on time...

Comment (danberindei) [inline core/src/main/java/org/infinispan/distribution/group/PartitionerConsistentHash.java]: I added an `isReplicated()` method in `ConsistentHash` and removed the TODO.

**Comment by tristantarrant** [comment]
Pushed, thanks

**Pull request closed without merging**